### PR TITLE
Enable IPv6 and make the topology dual-stack

### PR DESCRIPTION
**Pull request by larseggert**

### CODE REVIEW

File: endpoint/setup.sh [hunk 1]
```diff
@@ -14,5 +14,15 @@ route add -net 193.167.0.0 netmask 255.255.0.0 gw $GATEWAY
 # delete unused route
 route del -net $UNNEEDED_ROUTE netmask 255.255.255.0
 
+# XXX this relies on the IPv6 address being second in the "hostname -I" output
```
Comment (janaiyengar) [inline endpoint/setup.sh:17]: Can you remove the `XXX` from the comment? It'd also be good to know if the output of `hostname` changed. Can you add an
```
echo "Endpoint's IPv6 address is $IP"
```
which can be helpful when it breaks? Same above for the v4 address, that also relies on the v4 address being the first in the output.

File: sim/scenarios/helper/quic-point-to-point-helper.cc [hunk 2]
```diff
@@ -22,5 +24,13 @@ NetDeviceContainer QuicPointToPointHelper::Install(Ptr<Node> a, Ptr<Node> b) {
   tch.SetRootQueueDisc("ns3::PfifoFastQueueDisc", "MaxSize", queue_size_);
   tch.Install(devices);
 
+  Ipv4AddressHelper ipv4;
+  ipv4.SetBase("193.167.50.0", "255.255.255.0");
+  ipv4.Assign(devices);
+
+  Ipv6AddressHelper ipv6;
+  ipv6.SetBase("fd00:cafe:cafe:50::", 64);
+  ipv6.Assign(devices);
```
Comment (janaiyengar) [inline sim/scenarios/helper/quic-point-to-point-helper.cc:33]: nice cleanup, thanks!

**Review by janaiyengar — COMMENTED** [review]
@larseggert : Are you ready for this to be reviewed? I haven't tested this, assuming that you've done that?

**Comment by larseggert** [comment]
Yes, this is ready for review

**Review by marten-seemann — APPROVED** [review]
This is great! Thank you @larseggert!

**Pull request merged**